C/plugins/storage/postgres/connection.cpp

```diff
@@ -2546,3 +2546,32 @@ void Connection::logSQL(const char *tag, const char *stmt)
 		Logger::getLogger()->info("%s: %s", tag, stmt);
 	}
 }
+
+/**
+ * Create snapshot of a common table
+ *
+ * @param table		The table to snapshot
+ * @param id		The snapshot id
+ * @return		-1 on error, >= 0 on success
+ *
+ * The new created table name has the name:
+ * table_id
+ */
+int Connection::create_table_snapshot(const string& table, const string& id)
+{
+	string query = "SELECT * INTO TABLE foglamp.";
+	query += table + "_" +  id + " FROM foglamp.configuration";
```

Comment:
Should be table, not "foglamp.confoiguration"

Reply:
Yes, oversight!